lte/gateway/python/magma/pipelined/app/classifier.py

```diff
@@ -719,6 +719,7 @@ def gtp_handler(
             session_state == PdrState.Value('INSTALL')
             or session_state == UESessionState.ACTIVE
         ):
+            self.remove_paging_flow(ue_ip_addr)
```

Comment:
Is there need to remove this flows, since the paging flow and active flow match part is same that flow would be overwritten.

Reply:
Yes, because we are using paging priority (5) for paging flow. Hence the flow could not be overwritten.

Reply:
why the priority is changed ?
can we keep it same as the active session flow?

Reply:
Default paging priority is 5 as per 4G design(OAI code).
For active session flow priority will calculate based on coming precedence value from sessiond.

Reply:
But even for downlink session flow is present in_port=LOCAL/mtr0. So it could not be overwritten the flow.

Reply:
ok, we dont have to follow same priorities as 4G flow table.
Lets add TODO to unblock this PR.